feat: add expo-clipboard library (#11227)
expo/expo: packages/expo-clipboard/README.md

# expo-clipboard

> `expo-clipboard` is a drop-in replacement for React Native's Clipboard (which is now deprecated). This library gives you an interface for getting and setting the content from your user's Clipboard on Android, iOS, and Web.

Reviewer: ```suggestion
**`expo-clipboard`** provides an interface for getting and setting the content from a user's Clipboard on Android, iOS, and Web.
```

might as well just use the same description as in docs

Reviewer: (then we don't have to update this description text if we change the api so it's not a drop-in replacement anymore or to clarify the nuance of actually it's not exactly a drop-in replacement, there is a slightly different api because of the async suffix)
